src/pages/basic_data/institution_admin/UserRolesSelector.tsx

```diff
@@ -91,6 +91,15 @@ export const UserRolesSelector = ({
                 />
               }
             />
```

Comment:
Legger NVI-rollen inn i boksen for App-admin, slik at dette ikke blir fritt frem for alle å teste før det er klart. Men nå kan sikt-ansatte få denne rollen via frontend.